components/admin_console/system_user_detail/system_user_detail.tsx

```diff
@@ -68,99 +91,100 @@ export default class SystemUserDetail extends React.PureComponent {
             customComponentWrapperClass: '',
             user: {
                 email: this.props.user.email,
-            },
+            } as UserProfile,
             addTeamOpen: false,
             refreshTeams: true,
+            error: '',
         };
     }
 
-    setTeamsData = (teams) => {
-        const teamIds = teams.map((team) => team.team_id);
+    setTeamsData = (teams: Record<string, any>): void => {
```

Comment:
Shouldn't it be Record<string, Team>?

Reply:
This function was required in `team_list.tsx` so I assumed it had  the correct type and added it here, turns out that wasn't the case, updated both places with `GroupTeam[]` type